ansible/roles/operator-pipeline/tasks/community-release-pipeline-trigger.yml

```diff
@@ -117,3 +134,12 @@
                     - name: registry-credentials
```

Comment:
The workspace that you left here is not present in the isv release pipeline. Please update the isv release pipeline in the way it supports this workpace or remove it if it is not needed and can be replaced by a different workspace.